.github/workflows/wheel.yml

```
@@ -32,69 +32,125 @@ on:
     types:
       - published
 jobs:
-  generate-wheels-matrix:
-    # https://iscinumpy.dev/post/cibuildwheel-2-10-0/
-    name: Generate wheels matrix
+  generate-linux-wheels-matrix:
+    name: Generate Linux wheel matrix
     runs-on: ubuntu-latest
     outputs:
       include: ${{ steps.set-matrix.outputs.include }}
     steps:
       - uses: actions/checkout@v3
       - name: Install cibuildwheel
-        run: pipx install cibuildwheel==2.16.2
+        run: pipx install cibuildwheel==2.22.0
       - id: set-matrix
         run: |
           MATRIX=$(
             {
               cibuildwheel --print-build-identifiers --platform linux \
-              | jq -nRc '{"only": inputs, "os": "ubuntu-latest"}'
-            } | jq -sc
-          )
+              | jq -nRc '{"only": inputs, "os": "ubuntu-latest", "toolset": "gcc"}' 
+            } | jq -sc)
```

nit:
```suggestion
              | jq -nRc '{"only": inputs, "os": "ubuntu-latest", "toolset": "gcc"}' 
            } | jq -sc
            )
```
With paren placed on new line diff will be shorter